_includes/components/head.njk

```diff
@@ -24,7 +24,7 @@
   {% endset %}
   <style>{{ css | cssmin | safe }}</style>
   {% set js %}
-    {# {% include "assets/js/jquery-3.4.1.min.js" %} #}
+    {% include "assets/js/jquery-3.4.1.min.js" %}
```

Comment:
Do we not need jquery anymore? 🤘

Reply:
If so, we can just delete this line

Reply:
This is outdated ^^^ please see recent commit. We do need jquery.